js_modules/dagster-ui/packages/ui-core/src/graph/SVGViewport.tsx

```diff
@@ -78,7 +79,7 @@ const PanAndZoomInteractor: SVGViewportInteractor = {
     let lastY: number = start.y;
     const travel = {x: 0, y: 0};
 
-    const onMove = (e: MouseEvent) => {
+    const onMove = throttle((e: MouseEvent) => {
```

Comment:
This seems like a good add, and it's pretty elegant we can one-line it! It's weird these get fired so fast though - it seems crazy the browser would allow them to arrive at more than 60fps. It makes me wonder if we're accidentally calling `document.addEventListener('mousemove', onMove);` N times or something weird like that, or removing it only sometimes, causing every event to be delivered multiple times once some thing has occurred...

Maybe before we merge we could sanity check and make sure that the `e.offsetX, e.offsetY` values in this function aren't repeating? Maybe each time you do a pan gesture, something doesn't get cleaned up, and it repeats one more time. 🙃 

Or it's possible that this is actually a hot-reloading issue? Maybe the event handlers get doubled up in dev mode somehow? 

Probably not worth a ton of investigation, but if this works I think every `move` event handler I've ever written needs a `throttle`!